include/ppx/application.h

```diff
@@ -209,8 +209,25 @@ struct ApplicationSettings
     bool enableDisplay         = true;
     bool enableImGui           = false;
     bool allowThirdPartyAssets = false;
-    bool enableXR              = false;
-    bool enableXRDebugCapture  = false;
+
+    struct
+    {
+        bool enable             = false;
```

Comment:
can you also add an option here to disable depth swapchain as well? this is for easier testing the perf impact of with and without the depth swapchain since it is not required by some reprojection techniques.